google/cloud/bigtable/tests/admin_integration_test.cc

```diff
@@ -107,7 +107,9 @@ TEST_F(AdminIntegrationTest, DropRowsByPrefixTest) {
        {column_family2, bigtable::GcRule::MaxNumVersions(10)},
        {column_family3, bigtable::GcRule::MaxNumVersions(10)}},
       {});
-  auto table = CreateTable(table_id, table_config);
+  auto table_meta = table_admin_->CreateTable(table_id, table_config);
```

Comment:
I think this test could use `GetTable()`

Reply:
Done.